### DISCUSSION TITLE
Unable to create account on FastMRI

### DISCUSSION THREAD

**Discussion by vigsivan** [facebookresearch/fastMRI#237] [General]
Hello,

I am having trouble with creating an account on the FASTMRI website. When I try to create an account, I get a message. 

"Processing, please wait..."

I have tried creating accounts from different browsers to no avail. Can you confirm if this is an issue on the website?

Thanks.

**Comment by liuyu98tju** [comment]
Yes, I meet the same problem with you

**Comment by mmuckley** [comment]
Hello @vigsivan, could you try again now? I just fixed a database connection issue.

**Reply by vigsivan** [reply]
Yes, thanks it works now! 💯

**Comment by liuyu98tju** [comment]
Yes! Thank you so much! It's so nice of you. Glad to see you.<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body>
<br/><br/><br/><div class="ntes-mailmaster-quote" style="padding-top: 1px; padding-bottom: 1px" >
    <div style=" margin-top: 2em; margin-bottom: 1em; font-size: 14px; line-height: 1.25; color: #78787a; " >---- 回复的原邮件 ----</div>
    <div style=" margin-bottom: 1em; font-size: 12px; line-height: 1.25; color: #232324; padding: 0.5em 0.25em; border-radius: 0.67em; background-color: #f0f0f0; " >
        <table width="100%" cellpadding="0" cellspacing="9" border="0">
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    发件人
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-from" style="text-decoration:none;color:#0886e8;" ***@***.***">Matthew ***@***.***&gt;</a>
                </td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    日期
                </td>
                <td class="mail-date" valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    2022年04月13日 22:03
                </td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    收件人
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-to" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr style="">
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    抄送至
                </td>
                <td valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    <a class="mail-cc" style="text-decoration:none;color:#0886e8;" ***@***.***">Yu ***@***.***&gt;</a>、<a class="mail-cc" style="text-decoration:none;color:#0886e8;" ***@***.******@***.***&gt;</a>
                </td>
            </tr>
            <tr>
                <td valign="top" style=" width: 4.25em; font-size: 12px; line-height: 1.25; color: #78787a; " >
                    主题
                </td>
                <td class="mail-subject" valign="top" style=" font-size: 12px; line-height: 1.25; color: #232324; word-break: break-all; " >
                    Re: [facebookresearch/fastMRI] Unable to create account on FastMRI (Discussion #237)
                </td>
            </tr>
        </table>
    </div>
    <div><p></p>
<p dir="auto">Hello <a class="user-mention" data-hovercard-type="user" data-hovercard-url="/users/vigsivan/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" ***@***.***</a>, could you try again now? I just fixed a database connection issue.</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br />Reply to this email directly, <a href="https://github.com/facebookresearch/fastMRI/discussions/237#discussioncomment-2559958">view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/AL3ZY7ZIBLSPPJYFXSINLHLVE3H3FANCNFSM5TI6B3VQ">unsubscribe</a>.<br />You are receiving this because you commented.<img src="https://github.com/notifications/beacon/AL3ZY76MLKHJVHLVBWNJZQTVE3H3FA5CNFSM5TI6B3V2YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAATQ7VQ.gif" height="1" width="1" alt="" /><span style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message ID: <span>&lt;facebookresearch/fastMRI/repo-discussions/237/comments/2559958</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>
<script type="application/ld+json">[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/facebookresearch/fastMRI/discussions/237#discussioncomment-2559958",
"url": "https://github.com/facebookresearch/fastMRI/discussions/237#discussioncomment-2559958",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]</script>
</div>
</div>
</body>
</html>
